intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/IntelliJMarkupModelListener.java

```diff
@@ -0,0 +1,26 @@
+package nl.tudelft.watchdog.intellij.logic.ui.listeners;
+
+import com.intellij.openapi.editor.ex.RangeHighlighterEx;
+import com.intellij.openapi.editor.impl.event.MarkupModelListener;
+import nl.tudelft.watchdog.core.logic.event.TrackingEventManager;
+import org.jetbrains.annotations.NotNull;
+
+public class IntelliJMarkupModelListener implements MarkupModelListener {
```

Comment:
This should be part of another PR, right?

Reply:
I included it to show how a new class would fit in this model. It will be changed in the next PR, but I can remove it if you want to?

Reply:
Yeah, I think that would be cleaner. Thanks